src/brevitas/export/common/handler/qcdq.py

```diff
@@ -133,6 +183,120 @@ def dequantize_symbolic_kwargs(cls, scale, zero_point, bit_width, is_signed):
             'scale_orig_shape': scale_orig_shape}
 
 
+class QCDQCastFloatWeightQuantProxyHandlerMixin(FloatQMixin, FloatCDQCastProxyHandlerMixin):
```

Comment:
Nothing important but if we want to be coherent with names, I'd say FloatQCDQCast...